strapi/strapi
# enhancement: use defaultLocale on delete/publish/unpublish/discardDra…

### What does it do?

use defaultLocale on delete/publish/unpublish/discardDraft by default, add * for all

### Why is it needed?

To make the API consistent (working on a single locale at a time by default)

### How to test it?

- Programatically with strapi.documents(uid)....
- using the contentAPI check the delete endpoint will only delete the default locale or the provided locale & supports the `*` param to delete all of them at once

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
